runtime/src/main/java/org/corfudb/runtime/exceptions/TrimmedException.java

```diff
@@ -24,7 +25,7 @@
      */
     @Getter
     @Setter
-    private List<Long> trimmedAddresses;
+    private List<Long> trimmedAddresses = Collections.emptyList();
```

Comment:
I think its cleaner to make this field final and assign the empty list in the constructors that don't need it.

Reply:
Done.